pages/texteditor.tsx

```diff
@@ -1,10 +1,39 @@
 ﻿import React from "react";
 import {NextPage} from "next";
-import TextEditorContainer from "../components/TextEditorContainer";
+import TextEditorContainer from "../components/TextEditor/TextEditorContainer";
+import noAutoComplete from "../components/TextEditor/Settings/NoAutoComplete";
+import limitedAutoComplete from "../components/TextEditor/Settings/LimitedAutoComplete";
+import fullAutoComplete from "../components/TextEditor/Settings/FullAutoComplete";
 
 const TextEditor: NextPage = () => <div>
-    <h1>This is the Text Editor</h1>
-    <TextEditorContainer/>
+    <h1>These are the different Text Editors</h1>
+    <h2>A Text Editor with no highlighting or autocomplete</h2>
+    <TextEditorContainer containerClassName="textEditor"
+                         theme="dark"
+                         height="90vh"
+                         width="90vw"
+                         language="unset"
```

Comment:
why is the language unset here?

Reply:
basically it is the only way to get the text editor to stop giving hints on the code